src/Sulu/Component/Rest/ListBuilder/Metadata/ListXmlLoader.php

```diff
@@ -126,12 +126,23 @@ private function loadPropertyMetadata(\DOMXPath $xpath, \DOMNode $propertyNode)
         $propertyMetadata->setSortable(
             XmlUtil::getBooleanValueFromXPath('@sortable', $xpath, $propertyNode, true)
         );
+
+        $parameters = $xpath->query('x:params', $propertyNode);
+        if (\count($parameters) > 0) {
+            $propertyMetadata->setParameters(
+                $this->getParameters(
+                    $xpath,
+                    $parameters->item(0) // There can only be one filter node
```

Comment:
Wrong comment, "There can onl ybe one params node", resp. we should wait on the decision about the suggestion I just posted.